test/dialogflow/DialogFlowEvent.spec.ts

```diff
@@ -177,6 +177,36 @@ describe("DialogFlowEvent", () => {
   });
 });
 
+describe("Facebook Messenger", () => {
```

Comment:
please rework this test, seems more like a test for the entire voxa flow than for just the DialogflowEvent, compare it to the other tests in the same module, most of them just initialize the object and then make expectations on the object values/methods

https://github.com/VoxaAI/voxa/blob/1e39c38a964bda48bf29f58657b9e27763e4181d/test/dialogflow/DialogFlowEvent.spec.ts#L105